File: Client/WebClient/src/app/models/dtos/port/attachOtauDto.ts
import { NetAddress } from "../../underlying/netAddress";
import { RtuMaker } from "../../enums/rtuMaker";

export class AttachOtauDto {
  ConnectionId: string;
  ClientIp: string;

  OtauId: string;
  RtuId: string;
  RtuMaker: RtuMaker;

  NetAddress: NetAddress;
  OpticalPort: number;
}

File: Shared/Dto/C2D2R/AttachOtauDto.cs
﻿using System;
using System.Runtime.Serialization;

namespace Iit.Fibertest.Dto
{
    [DataContract]
    public class AttachOtauDto
    {
        [DataMember]
        public string ClientIp { get; set; }

        [DataMember]
        public Guid RtuId { get; set; }

        [DataMember]
        public RtuMaker RtuMaker { get; set; }

        [DataMember]
        public Guid OtauId { get; set; }

        [DataMember]
        public NetAddress NetAddress { get; set; }

        [DataMember]
        public int OpticalPort { get; set; }
    }
  
}